### PR TITLE
Add typography constants and mixins.

### PR DESCRIPTION
**Pull request by chuckharmston**
r? @ngokevin

### CODE REVIEW

File: src/media/css/base.styl [hunk 1]
```diff
@@ -1,4 +1,4 @@
-@import 'constants';
+@import 'constants/index';
```
Comment (ngokevin) [inline src/media/css/base.styl:1]: i don't think this is necessary

Comment (chuckharmston) [inline src/media/css/base.styl:1]: It's not. It's an artifact from testing the bug I fixed in 33cabc8. Good catch!

File: src/media/css/constants/typography.styl [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+ *
+ * As much as is reasonable, text styles should be contained in mixins for
+ * consistency, reusability, and a clear sign of intent. These mixins should
+ * be defined either here or in a project-specific typography file.
+ *
+ * Type style mixin names should follow the form type--style-name
+ */
+
+// Font stacks
+$font-stack--fira-sans = "Fira Sans OT", "Fira Sans", FiraSansWeb, sans-serif;
```
Comment (ngokevin) [inline src/media/css/constants/typography.styl:24]: font-stack--primary and font-stack--mono to keep it agnostic?

Comment (chuckharmston) [inline src/media/css/constants/typography.styl:24]: Probably `sans` and `mono`, but yeah, you're right.

**Pull request merged**